trillian_admin_api.proto

```diff
@@ -21,6 +21,7 @@ option java_outer_classname = "TrillianApiProto";
 package trillian;
 
 import "trillian.proto";
```

Comment:
Update2: aaaaand this broke Travis. It's not as simple as I thought. Sorry about that, please revert.